README.md

```diff
@@ -108,6 +108,8 @@ Creating artifacts/abis/ to store compilation artifacts
 
 > NOTICE: this method doesn't use an account, which will be deprecated very soon as StarkNet makes deployments from accounts mandatory.
 
+> Token for deployments to Alpha MainNet can be set with the `--token` option.
```

Comment:
```suggestion
> Token for deployments to Alpha Mainnet can be set with the `--token` option.
```
It looks a little funny with the capital _-N_, no?

Reply:
Agree. I used this because I found it like this in Starknet documentation, but I agree it looks funny :).